hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMRatisResponse.java

```diff
@@ -87,14 +86,13 @@ public static SCMRatisResponse decode(RaftClientReply reply)
       return new SCMRatisResponse(reply.getException());
     }
 
-    final byte[] response = reply.getMessage().getContent().toByteArray();
+    final ByteString response = reply.getMessage().getContent();
 
-    if (response.length == 0) {
+    if (response.isEmpty()) {
       return new SCMRatisResponse();
     }
 
-    final SCMRatisResponseProto responseProto = SCMRatisResponseProto
-        .parseFrom(response);
+    final SCMRatisResponseProto responseProto = SCMRatisResponseProto.parseFrom(response.toByteArray());
```

Comment:
Can use the below to avoid creating new buffer too.
```
final SCMRatisResponseProto responseProto = SCMRatisResponseProto.parseFrom(response.newInput());
```

Reply:
@duongkame , thanks for reviewing this! 

The method `parseFrom(InputStream)` will call `new CodedInputStream(InputStream)`, which creates a `byte[]` in the `CodedInputStream` constructor.  Since it is `InputStream`, it probably will copy at least two times.